plugins/inventory/openstack.py

```diff
@@ -51,11 +51,14 @@
 class OpenStackInventory(object):
 
     def __init__(self, private=False, refresh=False):
+        config_files = os_client_config.config.CONFIG_FILES
+        if os.environ.get('OPENSTACK_CONFIG_FILE', None):
+            config_files.insert(0, os.environ['OPENSTACK_CONFIG_FILE'])
```

Comment:
Would you be amenable to changing this to "OS_CLIENT_CONFIG_FILE"? If so, I'd like to upstream support for setting this into os-client-config itself (it seems like a generally useful feature) We can land it here safely (with the name change) until we get it landed there.